backend/database/repositories/main.go

```diff
@@ -28,10 +30,6 @@ func GetRepository(repo int) interface{} {
 		return filesystemRepository{
 			embeddedContext{context},
 		}
-	case PERSON:
-		return personRepository{
-			embeddedContext{context},
-		}
```

Comment:
You removed this from the GetRepository function however the dependency_factor still attempts to instantiate it (line 31 of `dependency_factory.go`), please remove that reference otherwise we could very easilly have a nil pointer dereference